lightning/src/ln/channel.rs

```diff
@@ -12240,82 +12221,39 @@ mod tests {
 		assert!(node_a_chan.check_get_channel_ready(0, &&logger).is_some());
 	}
 
-	fn funding_input_sats(input_value_sats: u64) -> (TxIn, TransactionU16LenLimited) {
```

Comment:
CI is failing with unused imports for these two types.

Reply:
Fixed